### PR TITLE
TN: Allow date-only event times

### PR DESCRIPTION
**Pull request by mileswwatkins**

### CODE REVIEW

File: openstates/tn/events.py [hunk 1]
```diff
@@ -95,17 +95,24 @@ def scrape(self, chamber, session):
                 description = metainf['type'].text_content()
 
                 dtfmt = "%A, %B %d, %Y %I:%M %p"
+                dtfmt_no_time = "%A, %B %d, %Y"
                 if time == 'Cancelled':
                     self.log("Skipping cancelled event.")
                     continue
                 else:
+                    print(datetime_string)
```
Comment (rshorey) [inline openstates/tn/events.py]: remove print statement?

Comment (paultag) [inline openstates/tn/events.py]: :+1:

Comment (mileswwatkins) [inline openstates/tn/events.py]: Heh.

**Comment by rshorey** [comment]
probably a pre-existing issue, but it looks like for documents name and url are reversed. For example I'm getting:

    "documents": [
        {
            "name": "http://wapp.capitol.tn.gov/apps/Calendar/CalendarMain.aspx?committeeid=980000&calendardate=4/7/2015&ga=109",
            "type": "agenda",
            "url": "Agenda"
        }
    ]

**Comment by paultag** [comment]
we should fix that up, nice catch!

**Comment by mileswwatkins** [comment]
Alright, changed from positional to named parameters!

**Comment by rshorey** [comment]
looks good, thanks!

**Pull request merged**